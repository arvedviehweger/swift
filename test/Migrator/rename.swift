// REQUIRES: objc_interop
// RUN: rm -rf %t && mkdir -p %t && %target-swift-frontend -c -update-code -primary-file %s -F %S/mock-sdk -api-diff-data-file %S/API.json -emit-migrated-file-path %t/rename.swift.result -emit-remap-file-path %t/rename.swift.remap -o /dev/null
// RUN: diff -u %S/rename.swift.expected %t/rename.swift.result

import Bar

func foo(_ b: BarForwardDeclaredClass) {
  b.barInstanceFunc1(_: 0, anotherValue: 1, anotherValue1: 2, anotherValue2: 3)
  barGlobalFuncOldName(2)
  b.barInstanceFunc1(_: 0, anotherValue: 1, anotherValue1: 2, anotherValue2: 3)
  barGlobalFuncOldName(2)
  b.barInstanceFunc1(_: 0, anotherValue: 1, anotherValue1: 2, anotherValue2: 3)
  barGlobalFuncOldName(2)
  b.barInstanceFunc1(_: 0, anotherValue: 1, anotherValue1: 2, anotherValue2: 3)
  barGlobalFuncOldName(2)
  b.barInstanceFunc1(_: 0, anotherValue: 1, anotherValue1: 2, anotherValue2: 3)
  barGlobalFuncOldName(2)
  _ = barGlobalVariableOldEnumElement
}
