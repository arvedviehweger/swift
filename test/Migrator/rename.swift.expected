// REQUIRES: objc_interop
// RUN: rm -rf %t && mkdir -p %t && %target-swift-frontend -c -update-code -primary-file %s -F %S/mock-sdk -api-diff-data-file %S/API.json -emit-migrated-file-path %t/rename.swift.result -emit-remap-file-path %t/rename.swift.remap -o /dev/null
// RUN: diff -u %S/rename.swift.expected %t/rename.swift.result

import Bar

func foo(_ b: BarForwardDeclaredClass) {
  b.barNewInstanceFunc1(_: 0, newlabel2: 1, newlabel3: 2, newlabel4: 3)
  barGlobalFuncNewName(2)
  b.barNewInstanceFunc1(_: 0, newlabel2: 1, newlabel3: 2, newlabel4: 3)
  barGlobalFuncNewName(2)
  b.barNewInstanceFunc1(_: 0, newlabel2: 1, newlabel3: 2, newlabel4: 3)
  barGlobalFuncNewName(2)
  b.barNewInstanceFunc1(_: 0, newlabel2: 1, newlabel3: 2, newlabel4: 3)
  barGlobalFuncNewName(2)
  b.barNewInstanceFunc1(_: 0, newlabel2: 1, newlabel3: 2, newlabel4: 3)
  barGlobalFuncNewName(2)
  _ = NewEnum.enumElement
}
